slices/squashfs-tools.yaml

```diff
@@ -0,0 +1,25 @@
+package: squashfs-tools
+
+essential:
+  - squashfs-tools_copyright
+
+slices:
+  bins:
+    # adding libgcc-s1 as mksquashfs crashes without
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
```

Comment:
Do we need `libgcc-s1_libs`? Im not seeing it on apt show.

```
apt show $_
Package: squashfs-tools
Version: 1:4.6.1-1build1
...
Depends: libc6 (>= 2.38), liblz4-1 (>= 0.0~r130), liblzma5 (>= 5.1.1alpha+20120614), liblzo2-2 (>= 2.02), libzstd1 (>= 1.5.5), zlib1g (>= 1:1.1.4)
...
```

Reply:
It's odd, as I mention in my comment squashfs crashes without

Reply:
Looks `libgcc-s1` is a depndancy of libc6. Could you add to the libc6_libs essentials?

```
ubuntu@noble:~$ apt show libc6
Package: libc6
Version: 2.39-0ubuntu8.3
...
Depends: libgcc-s1

```

Reply:
I can't, this will cause a cyclic dependency between libgcc-s1 and libc6, that's why there are no dependencies between those two

Reply:
That is a problem! 

@rebornplusplus Are we aware of this issue? I'll make a bug ticket if not. Looks like apt supports cyclic dependencies.  


```

$ apt show libc6 | grep Depends
...
Depends: libgcc-s1, libcrypt1 (>= 1:4.4.10-10ubuntu4)
...

$ apt show libgcc-s1 | grep Depends
...
Depends: gcc-12-base (= 12.3.0-1ubuntu1~22.04), libc6 (>= 2.35)
...

# patching libc6_libs to represent this results in an error.

$ chisel cut --root="./rootfs" --release="../" libc6_libs
2024/09/09 15:27:10 Processing ../ release...
error: essential loop detected: libc6_libs, libgcc-s1_libs
```

Reply:
I brought the cyclic dependency issue up in a meeting today; it is a known problem. Our solution is what you had initially by explicitly including libgcc-s1_libs. Apologies for the confusion.

Reply:
> it is a known problem

Is it documented anywhere btw? I had encountered it about a week ago when browsing slices here:

https://github.com/canonical/chisel-releases/blob/d919a1b48c5537b662a7257f3feb963d13323a57/slices/libpython3.12-stdlib.yaml#L12-L24

Reply:
Thanks @polarathene! Unfortunately, it is not listed anywhere proper. We should add those comments in `libc6` slices too. And yes, it would indeed be nice to add those comments here too @Meulengracht.

Reply:
I confirm this is known and that for now, in chisel-releases, we go `libgcc0s1->libc6` and not the other way around, as libc6 seems to be more globally required than the other.